modules/catalog/src/test/java/org/apache/ignite/internal/catalog/commands/DropTableCommandValidationTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.catalog.commands;
+
+import org.apache.ignite.internal.catalog.Catalog;
+import org.apache.ignite.internal.catalog.CatalogCommand;
+import org.apache.ignite.internal.catalog.CatalogValidationException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+/**
+ * Tests to verify validation of {@link DropTableCommand}.
+ */
+public class DropTableCommandValidationTest extends AbstractCommandValidationTest {
+    @ParameterizedTest(name = "[{index}] ''{argumentsWithNames}''")
+    @MethodSource("nullAndBlankStrings")
+    void schemaNameMustNotBeNullOrBlank(String name) {
+        DropTableCommandBuilder builder = DropTableCommand.builder();
+
+        builder.tableName("TEST")
+                .schemaName(name);
+
+        assertThrows(
+                builder::build,
+                CatalogValidationException.class,
+                "Name of the schema can't be null or blank"
```

Comment:
Are you shure that test exception message? We have a few utility classes for the purpose.

Reply:
not sure I understand the question... If you are asking does this test validates the message as well, then yes, it does

Reply:
We already have few utillity methods for the same. Let's use existing instead of adding new one, e.g org.apache.ignite.internal.testframework.IgniteTestUtils#assertThrowsWithCause

Reply:
fixed. Replaced custom assertThrows with `IgniteTestUtils#assertThrowsWithCause`